src/mixins/files.ts

```diff
@@ -148,11 +148,18 @@ export default class FilesMixin extends Vue {
    * @returns The url for the requested file
    */
   async createFileUrl (filename: string, path: string) {
-    const token = (await authApi.getOneShot()).data.result
-
     const filepath = (path) ? `${path}/${filename}` : `${filename}`
+    let url = `${this.apiUrl}/server/files/${filepath}?date=${Date.now()}`
+
+    const forceLogins = this.$store.getters['server/getConfig'].authorization.force_logins
+    const isTrusted = this.$store.getters['auth/getCurrentUser']?.username === '_TRUSTED_USER_'
```

Comment:
We also need to check for `_API__API_KEY_USER_USER_` here? Seems we do in a few places elsewhere in the code!

Reply:
Hmm, I don't immediately see how the frontend could be logged in with the API key user.. In any case, this should be `_API_KEY_USER_` instead of `_API__API_KEY_USER_USER_`, so the existing code is wrong too

Reply:
Indeed, I'm not sure either why the API user shows on those other places... shouldn't be a big issue to ignore it here!